ros2pkg/ros2pkg/resource/ament_cmake/package.xml.em

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>@package_name</name>
+  <version>0.0.0</version>
+  <description>Description of @package_name</description>
```

Comment:
Should this include the description provided by the user and fall back to this if None provided? (maybe this is planned for a follow up PR)